Scripts/AlphabetizeStrings.swift

```diff
@@ -0,0 +1,80 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Foundation
+
+// Look for all Localizable.strings files in Translations directory
+let rootURL = URL(fileURLWithPath: "./Translations")
+
+// Exit early if the Translations directory does not exist.
+guard FileManager.default.fileExists(atPath: rootURL.path) else {
+    print("Error: Translations directory not found.")
+    exit(1)
+}
+
+// Enumerate through the Translations directory to find all .strings files.
+guard let enumerator = FileManager.default.enumerator(at: rootURL, includingPropertiesForKeys: nil) else {
+    print("Error: Failed to enumerate Translations directory.")
+    exit(1)
+}
+
+// All .strings files in Translations directory
+var files = [URL]()
+
+// Collect all files with the .strings extension found in ./Translations.
+for case let fileURL as URL in enumerator {
+    if fileURL.pathExtension == "strings" {
+        files.append(fileURL)
+    }
+}
+
+// This regular expression pattern matches lines of the format:
+// "Key" = "Value";
+let regex = #/^\"(?<key>[^\"]+)\"\s*=\s*\"(?<value>[^\"]+)\";/#
+
+// Process each .strings file found.
+for fileURL in files {
+    // Attempt to read the file content.
+    guard let content = try? String(contentsOf: fileURL, encoding: .utf8) else {
```

Comment:
This can be an issue since it only picks up the English file. The file is encoded differently than all of the other language files and frankly I haven't investigated why it needs to be, which was done in https://github.com/jellyfin/Swiftfin/pull/934. Regardless, we shouldn't do a loop and instead explicitly get the English file if we only need to sort that single file.

Reply:
Gotcha! I just made this change. So the top half is pretty much all replaced by just:

`var fileURL = URL(fileURLWithPath: "./Translations/en.lproj/Localizable.strings")`

Any issue specifying the file like that?

Reply:
Also, do we want logic to remove unused localizations as part of the build? I'd like to think we can just avoid creating unused strings in the future but it would be nice for it to just be automated in case it does happen. On the other hand, it'd get annoying to lose localizations that are temporarily used that we might want later like `Replace`.

If we do want to purge unused strings, should that be a separate `PurgeStrings.swift` file or make one `OrganizeLocalizations.swift`?